totalRP3/Modules/NamePlates/NamePlates_Plater.lua

```diff
@@ -0,0 +1,90 @@
+
+local TRP3_PlaterNamePlates = {} -- magic public table
```

Comment:
May as well just drop the `local` here and not export it via `_G` later on - makes it easier to reason about on first-sight of this table.